src/features/__tests__/hometest.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly do not disable linters there helps to make your code look better

Reply:
changes made.